src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml.cs

```diff
@@ -351,12 +365,13 @@ private void OnMoreInfoClicked(object sender, RoutedEventArgs e)
 
         internal void CloseAutocompletionWindow(object sender, RoutedEventArgs e)
         {
-            OnRequestShowNodeAutoCompleteSearch(ShowHideFlags.Hide);
+            CloseAutoCompletion();
         }
 
         internal void CloseAutoCompletion()
         {
             OnRequestShowNodeAutoCompleteSearch(ShowHideFlags.Hide);
+            ViewModel.OnNodeAutoCompleteWindowClosed();
```

Comment:
Looks like the ViewModel could be null here
https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/12457/testReport/Dynamo.Tests/WorkspaceOpeningTests/ClearingWorkspaceResetsPositionAndZoom/